ruby/extism.gemspec

```diff
@@ -27,12 +27,13 @@ Gem::Specification.new do |spec|
       (f == __FILE__) || f.match(%r{\A(?:(?:bin|test|spec|features)/|\.(?:git|travis|circleci)|appveyor)})
     end
   end
+  spec.files.reject! { |f| f.end_with?(".wasm") }
   spec.bindir = "exe"
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
   # Uncomment to register a new dependency of your gem
-  spec.add_dependency "ffi", ">= 1.0.0"
+  spec.add_dependency "ffi", "~> 1.0"
```

Comment:
drive by commit, it's the same thing but `>=` is too liberal. This ensures we stay on 1.X but get updates.